### ISSUE TITLE
Python File Runs good but finishes/exit halfway and automatically ends command

### ISSUE THREAD

**Issue by securepdffileaccessapp** [novsession/reverseip#1]
Python File Runs good but finishes/exit halfway and automatically ends command. why is this? any solution?

**Comment by novsession** [comment]
yes, bro, I know, there are a number of things that should be closed, but instead print the error text

**Comment by novsession** [comment]
I've updated it. You can try again, don't forget to star this repository, good luck.

**Comment by Naughty-SEC** [comment]
is it limit for API on rapiddns? only 5 list of IP can be executed

**Comment by securepdffileaccessapp** [comment]
Yeah its working nice now but there are two errors I have noticed 

Error 1:
error :  ("Connection broken: InvalidChunkLength(got length b'', 0 bytes read)", InvalidChunkLength(got length b'', 0 bytes read))

Error 2:
error :  'charmap' codec can't encode characters in position 0-5: character maps to <undefined>


Kindly look into these thank you and thanks for your promptness in fixing the earlier issue as well. I will like to make a donation to the cause as well as soon as everything works fine.
